discovery_engine_core/bindings/.gitignore

```diff
@@ -1,4 +1,4 @@
 # Omit generated files
-# DELETE_AFTER_THIS_IN_RELEASE
+#NOTE if we want to publish this to crates.io auto generated fiels need some special care
```

Comment:
The build process would need to work without someone calling a `just` file.

So we need to make sure any generated files are included in the package, which they currently are not due to .gitignore.

For this we need to use `package.include` or `package.exclude` (which like .pubignore _fully_ replaces .gitignore wrt. package publishing).

Then we probably would also strip out the build.rs script, because when we ship it as a package there is no point in ad-hoc generating a c header which isn't used anyway (through we can bundle in the C header generated by CI if we want to).

We also probably then want to not push a Cargo.toml with the no longer needed build dependencies...

Anyway all of this is:
- way out of scope of what you should place into a comment in a .gitignore file
- irrelevant as hopefully we switched to the message passing API long before we consider seriously publishing this on crates.io, in which all the things are different anyway, preferably gone.

Reply:
I updated the comment.